jujugui/static/gui/src/app/init/charmstore-user.js

```diff
@@ -0,0 +1,58 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+'use strict';
+
+const CharmstoreUserMixin = (superclass) => class extends superclass {
```

Comment:
I will but I fully expect this mixin to disappear and this become part of the `user` class instead as this is excessive just for a charmstore user.